MaxText/maxengine.py

```diff
@@ -115,7 +111,7 @@ def model_apply(_p, _rng):
       params = {}
       params['aqt'] = new_vars['aqt']
       # Remove param values which have corresponding qtensors in aqt to save memory.
-      params['params'] = quantizations.remove_quantized_params(state.params, new_vars['aqt'])
+      params['params'] = quantizations.remove_quantized_params(state.params['params'], new_vars['aqt'])
```

Comment:
Alas, we need to be more explicit here now because we are specifically pruning one collection (params) with another (aqt); before `state.params` was just gonna be the parameters directly and the variable keys lined up, so we need to index into params to get that same consistency.